app.js

```diff
@@ -13,6 +13,28 @@ function hasFilter(name, filters) {
   return false;
 }
 
+function getParameterByName(name, url) {
+  if (!url) url = window.location.href;
```

Comment:
I would prefer getting the parameters from window.location.search rather than href.